src/main/java/Bob.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class Bob {
+
+    public static final String SEPARATOR = "____________________________________________________________";
+
+    public static void main(String[] args) {
+        String logo = "  ____        _     \n"
+                + " | |_) \\ ___ | |___  \n"
+                + " |  _ //  _  \\   _ \\ \n"
+                + " | |_)\\\\ (_) /  |_) |\n"
+                + " |____/ \\___/|_|___/ \n";
+        System.out.println(logo);
+        System.out.println("Hello! I'm Bob");
+        System.out.println("What can I do for you?");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String input = scanner.nextLine();
+            if (input.equals("bye")) {
+                System.out.println(SEPARATOR);
+                System.out.println("Bye. Hope to see you again soon!");
+                System.out.println(SEPARATOR);
+                break;
+            } else if (input.equals("list")) {
+                System.out.println(SEPARATOR);
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < taskCount; i++) {
+                    System.out.println((i+1) + "." + tasks[i]);
```

Comment:
Use spaces for readability: 
System.out.println((i + 1) + "." + tasks[i]);

Reply:
Great idea, thank you for your suggestion!